x-pack/plugins/enterprise_search/public/applications/analytics/components/layout/page_template.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { ENTERPRISE_SEARCH_CONTENT_PLUGIN } from '../../../../../common/constants';
+import { SetAnalyticsChrome } from '../../../shared/kibana_chrome';
+import {
+  EnterpriseSearchPageTemplateWrapper,
+  PageTemplateProps,
+  useEnterpriseSearchNav,
+} from '../../../shared/layout';
+import { SendEnterpriseSearchTelemetry } from '../../../shared/telemetry';
+
+export const EnterpriseSearchAnalyticsPageTemplate: React.FC<PageTemplateProps> = ({
```

Comment:
Not super clear why each top level section has its own page template and the code is identical. I've added it here to follow conventions